rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations.adoc

```diff
@@ -1,19 +1,23 @@
 
-
 = Excel Operations
 
-Excel Operations enable you to interact with Microsoft Excel files by extracting and manipulating their data.
+Excel operations enable you to interact with Microsoft Excel files by extracting and manipulating their data.
 
 * xref:toolbox-excel-operations-excel-controlled-loop.adoc[Excel Controlled Loop]
 +
-This Action Step enables you to iterate over data from an Excel file.
+Use Excel Controlled Loop to iterate over data from an Excel file.
 * xref:toolbox-excel-operations-excel-session.adoc[Excel Session]
 +
 The Excel Session is an element that encompasses all Excel operations. Excel operations include the reading, modification, and creation of entire files and file elements in the Excel format.
 * xref:toolbox-excel-operations-read-from-excel-file.adoc[Read from Excel File]
 +
-This Action Step enables you to read the contents of an Excel file.
+Use Read from Excel File to read the contents of an Excel file.
 * xref:toolbox-excel-operations-recalculate-excel-file.adoc[Recalculate Excel File]
 +
-This Action Step enables you to recalculate an Excel sheet
+Use Recalculate Excel File to recalculate the values of an Excel sheet.
+* xref:toolbox-excel-operations-save-excel-file.adoc[Save Excel File]
++
+Use Save Excel File to save the contents of an Excel file.
 * xref:toolbox-excel-operations-write-to-excel-file.adoc[Write to Excel File]
++
+Use Write to Excel File to write the contents from the read Excel file. You can write the content to the same file or to a new destination file.
```

Comment:
**suggestion**: Write to Excel file can be used to write any content to the excel file. The text reads as if only content that was read from the Read Excel File action step can be used.

```suggestion
Use Write to Excel File to write content to the selected excel file. You can write the content to the same file or to a new destination file.
```